src/StochasticStyles/spawning.jl

```diff
@@ -171,14 +171,14 @@ struct Exact{T} <: SpawningStrategy
     Exact(threshold::T=0.0) where {T} = new{T}(threshold)
 end
 
-@inline function spawn!(s::Exact, w, offdiags::AbstractVector, add, val, boost=1)
+@inline function spawn!(s::Exact, w, column, add, val, boost=1)
     T = valtype(w)
-    spawns = sum(offdiags; init=zero(T)) do (new_add, mat_elem)
+    spawns = sum(column; init=zero(T)) do (new_add, mat_elem)
```

Comment:
```suggestion
    spawns = sum(offdiagonals(column); init=zero(T)) do (new_add, mat_elem)
```
I disagree here with and think that `spawn!` should continue to deal with offdiagonals only.